src/bitwarden_exporter/bw_list_process.py

```
@@ -0,0 +1,277 @@
+    raw_items: Dict[str, Any] = {}
+    bw_current_status = json.loads(bw_exec(["status"], is_raw=False, settings=settings))
+    raw_items["status.json"] = bw_current_status
+
+    if bw_current_status["status"] == "unlocked":
+        settings.session_token = None
+
+    if bw_current_status["status"] != "unlocked":
+        raise BitwardenException("Vault is not unlocked")
+    LOGGER.debug("Vault status: %s", json.dumps(bw_current_status))
+
+    bw_folders_dict = json.loads((bw_exec(["list", "folders"], is_raw=False, settings=settings)))
```

Ruff (0.14.1)

Avoid extraneous parentheses

Remove extraneous parentheses

(UP034)